.travis.yml

```diff
@@ -29,8 +29,11 @@ notifications:
       on_success: change
       on_failure: always
 
-cache: 
-    - apt
+cache:
+  apt: true
+  ccache: true
+  directories:
+  - "$HOME/.ccache"
```

Comment:
There is bug on travis, if you use `ccache: true`, `ccache -s` will print zero results. Instead the suggested solution for now is to add the absolute ccache path.

This was mentioned in https://github.com/travis-ci/travis-ci/issues/3727